tribe-ext-event-progress-bar.php

```diff
@@ -38,29 +38,16 @@
 // Do not load unless Tribe Common is fully loaded and our class does not yet exist.
 if (
 	class_exists( 'Tribe__Extension' )
-	&& ! class_exists( NS . 'Main' )
+	//&& ! class_exists( NS . 'Main' )
```

Comment:
hrm, not sure why this is commented out? I think I did it but can't recall why.

Reply:
Done